build.gradle

```diff
@@ -26,7 +26,7 @@ ext {
   sizeofVersion = '0.3.0'
 
   // Clustered
-  terracottaPlatformVersion = '5.0.6.beta6'
+  terracottaPlatformVersion = '5.0.6.beta7'
```

Comment:
no because the previous commit is not compatible with 5.0.6.beta7.
The last single commit is to make ehcache compatible with 5.0.6.beta6